pkg/preflights/host-preflight.yaml

```diff
@@ -127,6 +127,21 @@ spec:
         collectorName: 'wildcard-check'
         hostnames:
           - '*'
+    - subnetAvailable:
+        collectorName: Pods Subnet
+        exclude: '{{ or (ne .GlobalCIDR.CIDR "")  (eq .PodCIDR.CIDR "") }}'
```

Comment:
`(ne .GlobalCIDR.CIDR "")` is unnecessary

Reply:
Done via - https://github.com/replicatedhq/embedded-cluster/pull/1355/commits/51a58ef43c7df77b23c5c10c820cf92fa0672dba